src/raspberryEsp32Interface.cpp

```diff
@@ -155,7 +155,7 @@ namespace bzzz
 
     bool RaspberryEsp32Interface::armed()
     {
-        return m_encodedSwitchesData & RADIO_SWITCH_B_BIT;
+        return m_encodedSwitchesData == RADIO_SWITCH_B_BIT && throttleReferencePercentage() < 0.05 ;
```

Comment:
`RADIO_SWITCH_B_BIT` is the *position* of the bits. I think this should be something like

```c++
bool isArmSwitchOn = m_encodedSwitchesData & RADIO_SWITCH_B_BIT;
bool isThrottleDown = throttleReferencePercentage() < 0.05;
return isArmSwitchOn && isThrottleDown;
```